spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializable.scala

```diff
@@ -285,6 +289,43 @@ trait ModuleSerializable extends Loadable with Savable{
    */
   protected def copy2BigDL[T: ClassTag](context: DeserializeContext, module : ModuleData[T])
                                        (implicit ev: TensorNumeric[T]): Unit = {
+
+    if (context.bigdlModule.getHasParameters) {
+      copyParameters2BigDL(context, module)
+    } else {
+      // for legacy format models
+      copyWeightAndBias(context, module)
```

Comment:
If we set `weight`, `bias` while `hasParameters` is `true`. What should we do?

Reply:
it could not happen, this `else` is only for those legacy format models already existing in model zoo